drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -153,6 +154,8 @@ int spi_nxp_init_common(const struct device *dev)
 		return err;
 	}
 
+	LPSPI_Reset(base);
```

Comment:
why was this needed ? just wondering.

Reply:
On the IMX95 we load code into ITCM and execute from there due to that there's no POR.
Thus we can't guarantee that the LPSPI in the right state on startup, so I prefer to reset a peripheral on initialization.

Reply:
ok, just FYI, I was planning to hardware reset peripheral at module level, in the PR I linked above, but we can do software peripheral reset also

Reply:
Well not every chip has a `RESET_ReleasePeripheralReset` interface whereas each LPSPI has a `LPSPI_Reset`.
So if you really want to use `RESET_ReleasePeripheralReset` you would have to implement a fallback anyhow.

Reply:
Yes I know, but actually the hardware signal reset is required on some platforms anyways before interfacing to the peripheral, even to do a software reset. But like I said I have no issue with this